configuration/pih/pih-config-mirebalais-humci.json

```diff
@@ -80,6 +80,53 @@
   },
 
   "localZlIdentifierGeneratorEnabled": "true",
-  "localZlIdentifierGeneratorPrefix": "Y"
+  "localZlIdentifierGeneratorPrefix": "Y",
+  "extensions": [
+    {
+      "id": "zl.report.hiv.powerbilink.dispensing.vih",
+      "type": "link",
+      "url": "https://app.powerbi.com/groups/me/apps/64103ba1-31cb-484a-bc61-b631946bfae7/reports/6e93ba97-b5f8-448f-bce4-760a8ded74f5/ReportSection?ctid=5254789f-6860-4375-85bc-302509fad508",
+      "extensionPointId": "org.openmrs.module.reportingui.reports.overview",
+      "label": "zl.report.hiv.powerbilink.dispensing",
+      "icon": "",
+      "requiredPrivilege": null
+    },
+    {
+      "id": "zl.report.hiv.powerbilink.viralCharge.vih",
+      "type": "link",
+      "url": "https://app.powerbi.com/groups/me/apps/64103ba1-31cb-484a-bc61-b631946bfae7/reports/6e93ba97-b5f8-448f-bce4-760a8ded74f5/ReportSection4cfdfea1bd30a40e2d59?ctid=5254789f-6860-4375-85bc-302509fad508",
+      "extensionPointId": "org.openmrs.module.reportingui.reports.overview",
+      "label": "zl.report.hiv.powerbilink.viralCharge",
+      "icon": "",
+      "requiredPrivilege": null
+    },
+    {
+      "id": "zl.report.hiv.powerbilink.preventionTb.vih",
+      "type": "link",
+      "url": "https://app.powerbi.com/groups/me/apps/64103ba1-31cb-484a-bc61-b631946bfae7/reports/6e93ba97-b5f8-448f-bce4-760a8ded74f5/ReportSection11f675956ec224840e0e?ctid=5254789f-6860-4375-85bc-302509fad508",
+      "extensionPointId": "org.openmrs.module.reportingui.reports.overview",
+      "label": "zl.report.hiv.powerbilink.preventionTb",
+      "icon": "",
+      "requiredPrivilege": null
+    },
+    {
+      "id": "zl.report.hiv.powerbilink.visitesClinique.vih",
+      "type": "link",
+      "url": "https://app.powerbi.com/groups/me/apps/64103ba1-31cb-484a-bc61-b631946bfae7/reports/6e93ba97-b5f8-448f-bce4-760a8ded74f5/ReportSectione685cdc7342305ea9a8b?ctid=5254789f-6860-4375-85bc-302509fad508",
+      "extensionPointId": "org.openmrs.module.reportingui.reports.overview",
+      "label": "zl.report.hiv.powerbilink.visitesClinique",
+      "icon": "",
+      "requiredPrivilege": null
+    },
+    {
+      "id": "zl.report.hiv.powerbilink.actifArv.vih",
+      "type": "link",
+      "url": "https://app.powerbi.com/groups/me/apps/64103ba1-31cb-484a-bc61-b631946bfae7/reports/6e93ba97-b5f8-448f-bce4-760a8ded74f5/ReportSection5c68520718adb108c099?ctid=5254789f-6860-4375-85bc-302509fad508",
+      "extensionPointId": "org.openmrs.module.reportingui.reports.overview",
+      "label": "zl.report.hiv.powerbilink.actifArv",
+      "icon": "",
+      "requiredPrivilege": null
+    }
+  ]
```

Comment:
I probably wouldn't include this in both files, now that I see it like this.  Do you need these to be present on humci?  Can we just include in the hiv configuration?  We can easily test on haitihivtest, and can also create a local SDK development server with the hiv configuration.

Reply:
I can remove it in humci. I don't think it is needed on humci. Let me update it.